WTWSMS/events/WtWSMS_migration_events.txt

```diff
@@ -236,14 +236,6 @@ character_event = {
 	
 	trigger = {
 		NOT = { year = 800 } # The migration period ended around 800
-		OR = { # Possible target religion groups
-			religion_group = christian
-			religion = finnish_pagan
-			religion = germanic_pagan
-			religion = baltic_pagan
-			religion = scythian_pagan
-			religion = slavic_pagan
-		}
```

Comment:
@loup99 / @EmperorEnlil 

Actually, I don't think it will!

It may only need lines in one file. Where it shows the AI target, you would put

AND = {
has_game_rule = {
name = MIGRATION
value = HISTORICAL
}
OR = {
(usual targets)
}
}